src/jit/gentree.cpp

```diff
@@ -911,6 +927,27 @@ Compiler::fgWalkResult Compiler::fgWalkTreePostRec(GenTreePtr* pTree, fgWalkData
             }
             break;
 
+        case GT_STORE_DYN_BLK:
+            result = fgWalkTreePostRec<computeStack>(&tree->gtDynBlk.Data(), fgWalkData);
+            if (result == WALK_ABORT)
+            {
+                return result;
+            }
+            __fallthrough;
+
+        case GT_DYN_BLK:
+            result = fgWalkTreePostRec<computeStack>(&tree->gtDynBlk.Addr(), fgWalkData);
+            if (result == WALK_ABORT)
+            {
+                return result;
+            }
+            result = fgWalkTreePostRec<computeStack>(&tree->gtDynBlk.gtDynamicSize, fgWalkData);
+            if (result == WALK_ABORT)
+            {
+                return result;
+            }
+            break;
+#if 0
         case GT_PHI:
```

Comment:
Hmm ... I don't recall making that change, or why I would have done so. It's just before the place where the old block ops were so maybe I had previously ifdef'd them out and the `#if 0` migrated. I'll remove the #if 0 (wonder why it passes all the tests, though?)

---
In reply to: [77257978](https://github.com/dotnet/coreclr/pull/7027#discussion_r77257978) [](ancestors = 77257978)